README.md

```diff
@@ -353,18 +354,6 @@ ______________________________________________________________________
 
 ______________________________________________________________________
 
-## Lightning Lite
```

Comment:
@awaelchli @carmocca @justusschock can we have fabric section here instead?
cc: @lantiga

Reply:
I'd leave it for now. We first release Fabric officially and then we can change in the 1.9.0 branch retrospectively